js/models/LocalSettings.js

```diff
@@ -16,7 +17,10 @@ export default class extends Model {
 
   defaults() {
     return {
-      macStyleWinControls: remote.process.platform === 'darwin',
+      windowControlStyle: remote.process.platform === 'darwin' ? 'mac' : 'win',
+      showAdvancedVisualEffects: true,
+      saveTransactionMetadata: true,
+      defaultTransactionFee: 'high',
```

Comment:
Can we please validate that any defaultTransactionFee provided is one of the available types (I assume 'high', 'medium', 'low')?